lib/cib/cib_utils.c

```diff
@@ -809,9 +809,9 @@ cib_read_config(GHashTable * options, xmlNode * current_cib)
 
     config = pcmk_find_cib_element(current_cib, PCMK_XE_CRM_CONFIG);
     if (config) {
-        pe_unpack_nvpairs(current_cib, config, PCMK_XE_CLUSTER_PROPERTY_SET,
-                          NULL, options, PCMK_VALUE_CIB_BOOTSTRAP_OPTIONS, TRUE,
-                          now, NULL);
+        pe_unpack_nvpairs(NULL, config, PCMK_XE_CLUSTER_PROPERTY_SET, NULL,
+                          options, PCMK_VALUE_CIB_BOOTSTRAP_OPTIONS, FALSE, now,
+                          NULL);
```

Comment:
Ah yes... I was only thinking about `sort_pairs()` at the time, which won't change the order. But a later same-scored block would still overwrite the first one.

We could reverse the order in case of equality with overwrite but that seems counterintuitive

Reply:
I don't think we want to preserve the original behavior, we want a consistent policy (bootstrap options processed first, then option blocks by score, then first-wins within a block or equally scored blocks)

Reply:
I'd like to document that policy in Pacemaker Explained, but there's no good place to do it currently. It could go in Cluster-Wide Configuration though it should mention it also applies to instance_attributes, meta_attributes, and utilization in other sections.